action_list:

  number_of_targets: 3

  action_0:
    action_type: pose    
    translation: [0.4, 0.2, 0.5]
    orientation: [0.0, 0.0, 0.0]
    duration: 3.0
     
  action_1:
    action_type: pose    
    translation: [0.4, 0.2, 0.26 ]
    orientation: [0.0, 0.0, 0.0]
    duration: 3.0
     
  action_2:
    action_type: pose    
    translation: [ 0.4, -0.3, 0.26]
    orientation: [ 0.0, 0.0, 0.0]
    duration: 0.5
